server/api/src/classifier.py

```diff
@@ -95,11 +123,12 @@ async def service_start(yagna_app: Yagna) -> None:
             package=package,
             max_workers=1,
             budget=10.0,
-            timeout=timedelta(minutes=30),
+            timeout=timedelta(minutes=99),
             subnet_tag=os.getenv("YAPAPI_SUBNET_TAG"),
             driver=os.getenv("YAPAPI_DRIVER", "zksync"),
             network=os.getenv("YAPAPI_NETWORK", "rinkeby"),
             event_consumer=log_summary(log_event_repr),
+            strategy=DenylistMS(os.getenv("DENY_LIST", "").split())
```

Comment:
Some comment/mention in README about how to use this `DENY_LIST` would be nice.

Reply:
https://github.com/golemfactory/galatea/pull/16/commits/29de452f1c19af2e5d0552582174871a86eda57b#diff-52af042a8f7f47fd511d97c15ec95d7b89eaa93428af8cc3227bc1112553f728